project/types.go

```diff
@@ -237,6 +237,7 @@ type Service interface {
 	DependentServices() []ServiceRelationship
 	Containers() ([]Container, error)
 	Scale(count int) error
+	SpecificiesHostPort() bool
```

Comment:
Should this really be part of the public interface? It seems like more of an internal helper function.